onnxruntime/core/framework/session_state.cc

```diff
@@ -16,6 +17,8 @@
 #include "core/framework/utils.h"
 #include "core/providers/cpu/controlflow/utils.h"
 #include "core/session/onnxruntime_session_options_config_keys.h"
+#include <boost/algorithm/string.hpp>
```

Comment:
><boost/algorithm/string.hpp> [](start = 9, length = 28)

I don't think we use boost. The binary size with boost templates may become uncontrollable.